config/manager/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ configMapGenerator:
 - files:
   - controller_manager_config.yaml
   name: manager-config
-apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I think this was accidental, probably the result of a coincidental bulk replace?